docs/news/news-v5.0.x.rst

```diff
@@ -0,0 +1,282 @@
+
+  - Added ``ERRORS_ABORT`` infrastructure.
+  - Added support for 'initial error handler'.
+  - Added error handling for 'unbound' errors to ``MPI_COMM_SELF``.
+  - Persistent collectives have been moved to the MPI namespace
+    from MPIX.
+  - Made ``MPI_Comm_get_info()``, ``MPI_File_get_info()``, and
+    ``MPI_Win_get_info()`` compliant to the standard.
+  - Added initial partitioned communication using persistent sends
+    and persistent receives.
+  - Added support for ``MPI_Info_get_string()``.
+  - Droped unknown/ignored info keys on communicators, files,
+    and windows.
+  - Added ``MPI_Isendrecv()`` and its variants.
+  - Added Support for ``MPI_Comm_idup_with_info()``.
+- Reworked how Open MPI integrates with 3rd party packages.
+  The decision was made to stop building 3rd-party packages
+  such as Libevent, HWLOC, PMIx, and PRRTE as MCA components
+  and instead 1) start relying on external libraries whenever
+  possible and 2) Open MPI builds the 3rd party libraries (if needed)
+  as independent libraries, rather than linked into libopen-pal.
+- Updated to use PMIx v4.2 branch - current hash: b59e49f
+- Updated to use PRRTE v2.1 branch - current hash: 8c09625
+- osc/rdma and various btls: Many fixes to better support btl/tcp.
+
+  - This patch series fixs many issues when running with
+    ``--mca osc rdma --mca btl tcp``, IE - TCP support for one sided
+    MPI calls.
+- Change the default component build behavior to prefer building
+  components as part of libmpi.so instead of individual DSOs.
+- Remove pml/yalla, mxm, mtl/psm, and ikrit components.
+- Remove all vestiges of the C/R support.
+- Various ROMIO v3.4.1 updates.
+- Use Pandoc to generate manpages
+- 32 bit atomics are now only supported via C11 compliant compilers.
+- Explicitly disable support for GNU gcc < v4.8.1 (note: the default
+  gcc compiler that is included in RHEL 7 is v4.8.5).
+- HWLOC: Require v1.11.0 or later.
+- HWLOC: Updated internal hwloc to v2.7.0.
+- HWLOC: enable --enable-plugins when appropriate
+  When running configure on the internal hwloc, enable plugins via
+  the --enable-plugins option when 1) --disable-dlopen was not
+  specified and 2) the user didn't add some variant of
+  --enable-plugins on the configure line already.  The motivation
+  for this change is to not pull in hwloc dependencies like libcuda
+  in the general case.
+- Do not build Open SHMEM layer when there are no SPMLs available.
+  Currently, this means the Open SHMEM layer will only build if
+  the UCX library is found.
+- Fix rank-by algorithms to properly rank by object and span.
+- Updated the ``-mca pml`` option to only accept one pml, not a list.
+- vprotocol/pessimist: Updated to support ``MPI_THREAD_MULLTIPLE``.
+- btl/tcp: Updated to use reachability and graph solving for global
+  interface matching. This has been shown to improve ``MPI_Init()``
+  performance under btl/tcp.
+- fs/ime: Fixed compilation errors due to missing header inclusion
```

Comment:
What is fs/ime?  Put in more neutral language here, describing whatever the thing was that was fixed.